README.md

```diff
@@ -18,6 +18,15 @@ of the pipeline description, and more.
  * **Mailing List**: image-builder@redhat.com
  * **Changelog**: <https://github.com/osbuild/osbuild/releases>
 
+### Principles
+
+1. OSBuild stages are never broken, only deprecated. The same manifest should always produce the same output.
+2. OSBuild stages should be explicit wherever possible instead of e.g. relying on the state of the tree.
```

Comment:
"wherever" or "whenever"?
whenever sounds more correct to me, but up to you.

Reply:
Agreed